spec/jobs/usps_auth_token_refresh_job_spec.rb

```diff
@@ -86,7 +86,7 @@
     end
 
     context 'auth request throws network error' do
-      [Faraday::TimeoutError, Faraday::ConnectionFailed].each do |err_class|
+      [Faraday::TimeoutError, Faraday::ServerError, Faraday::ConnectionFailed].each do |err_class|
```

Comment:
@zachmargolis I'm wondering how i can test this to confirm that #login-alarms wouldnt be alerted

Reply:
The alarms go off when the jobs have unhandled exceptions, so by handling the exception (and the `subject.perform` doesn't throw) then the alarms won't go off